### ISSUE TITLE
Detect lost connection

### ISSUE THREAD

**Issue by pbvie** [wercker/stern#55]
It would be great if the script would indicate when the stream of a log is broken. At the moment it's not possible to know whether there are no new log message or if the connection has been lost.

Like `error: unexpected EOF` when running `kubectl logs`

**Comment by mikesplain** [comment]
Agreed, I'm not sure if it's the same issue but when pods exit with errors stern loses the stream and does not reconnect.

**Comment by svarkey** [comment]
> It would be great if the script would indicate when the stream of a log is broken. At the moment it's not possible to know whether there are no new log message or if the connection has been lost.
> 
> Like `error: unexpected EOF` when running `kubectl logs`

Did you find an answer for your question?

**Comment by paulbdavis** [comment]
I think this should be handled [here](https://github.com/wercker/stern/blob/master/stern/tail.go#L124), but I'm not sure that the specific error would be detectable.
